src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityCreativeEnergy.java

```diff
@@ -39,10 +42,8 @@ public class MetaTileEntityCreativeEnergy extends MetaTileEntity implements IEne
     private boolean active = false;
     private boolean source = true;
 
-    private long lastEnergyOutputPerSec = 0;
-    private long lastEnergyInputPerSec = 0;
-    private long energyOutputPerSec = 0;
-    private long energyInputPerSec = 0;
+    private long lastEnergyIOPerSec = 0;
+    private long energyIOPerSec = 0;
```

Comment:
This whole implementation seems strange (note: old implementation was worse) as you are using two values. One for setting IO and second for showing it's value (which is one update behind - why?). Is there reason for that, am I missing something?

Reply:
`lastEnergyIOPerSec` is useful for checking if the energy has actually changed, which allows one to lower the number of server/client syncs significantly.

Reply:
More I look at the whole block it makes less sense...then more sense...then less sense.

I don't think this can get any better without way bigger rewrite because of way energy network works.

Reply:
Is there anything in particular that I should further address in this PR to make at least this label make more sense?

Reply:
2 vars are needed to remove fluctuations. Every second the `energyIOPerSec` is resetted to 0. And every tick something is added. You dont want to see that on the screen.

Reply:
I got the whole point of resetting `energyIOPerSec` and for emitting energy it could be done with just one update in `update` and if done via using setter then old value would be known and then it could be synchronized without additional value or multiple calls. But then there is whole second part which I did not see correctly from GitHub and that is for energy acceptance which can't be done without second value as it can be called multiple times in one tick.

Reply:
Regarding saving `energyIOPerSecLast` to NBT it doesn't makes sense without saving `energyIOPerSec` too. As on load you will show last value (correct value), but on update current will be 0 as it will not be yet calculated, so next time you will show zero (wrong value), then correctly calculated value from now on (correct value). But given that everything gets calculated every tick I don't think it is even worth it to save that value as user won't see it.

Reply:
To fix this, I've created a separate variable `ticks` that counts up a second from when the MTE is loaded, which should not update `energyIOPerSec` earlier than expected.

Reply:
I feel that we don't see things in same way, and this whole thing is getting overengineered.

Also after exiting and re-entering world it shows 0 and it won't change till anything in GUI is clicked.

Reply:
Since it's a very particular point in which this bug I fixed would occur, it probably isn't worth fixing; so I've reverted that previous fix.

Reply:
Understandable choice, let's leave it as it is.